Index.html

```diff
@@ -33,6 +34,154 @@ <h2 class="subtitulo">I&apos;am a Software Developer</h2>
         </ul>
         </div>
     </section>
+
+    <section class="page1">
+      <h1 class="tituloMR">My Recent Works</h1>
```

Comment:
- [ ] Please note that you should have only one `h1` in the whole of your document, kindly change this to any other heading tag(from `h2` to `h5`). This is SEO best practice. Kindly check [here](https://yoast.com/one-h1-heading-per-post/) for more information.